packaging/debian/changelog

```diff
@@ -1,3 +1,8 @@
+ -- Damani Norman <damani.norman@rubrik.com>  Fri, 4 Oct 2019 16:50:00 -0800
+rear (2.5-1.rubrik) stable; urgency=low
+
+  * Added support for Rubrik CDM
+
```

Comment:
@gdha 
what about such added changelog entries to packaging/debian/changelog
To me our current
https://github.com/rear/rear/blob/master/packaging/debian/changelog
looks rather generic and not meant to list such details?
Cf. https://github.com/rear/rear/pull/2249/files#r332986422

Reply:
Revered this back to the master copy per your guidance.